libraries/vmmanager/vm_backend.cpp

```diff
@@ -0,0 +1,54 @@
+
+#include <koinos/vmmanager/exceptions.hpp>
+#include <koinos/vmmanager/vm_backend.hpp>
+
+#include <koinos/vmmanager/eos/eos_vm_backend.hpp>
+#include <koinos/vmmanager/fizzy/fizzy_vm_backend.hpp>
+
+#include <cstdlib>
+
+namespace koinos::vmmanager {
+
+vm_backend::vm_backend() {}
+vm_backend::~vm_backend() {}
+
+std::vector< std::shared_ptr< vm_backend > > get_vm_backends()
+{
+   std::vector< std::shared_ptr< vm_backend > > result;
+
+   result.push_back( std::make_shared< vmmanager::eos::eos_vm_backend >() );
+   result.push_back( std::make_shared< vmmanager::fizzy::fizzy_vm_backend >() );
+
+   return result;
+}
+
+std::string default_vm_backend = "fizzy";
```

Comment:
We should `constexpr std::string default_vm_backend = "fizzy";` inside the `get_default_vm_backend_name()` implementation.

Reply:
You'll have to settle for a `const std::string` instead because `constexpr` gives me a compiler error:

```
/root/p/koinos/koinos-chain/libraries/vmmanager/vm_backend.cpp: In function ‘std::string koinos::vmmanager::get_default_vm_backend_name()’:
/root/p/koinos/koinos-chain/libraries/vmmanager/vm_backend.cpp:27:26: error: the type ‘const string’ {aka ‘const std::__cxx11::basic_string<char>’} of ‘constexpr’ variable ‘default_vm_backend’ is not literal
   27 |    constexpr std::string default_vm_backend = "fizzy";
      |                          ^~~~~~~~~~~~~~~~~~
In file included from /usr/include/c++/10/string:55,
                 from /root/.hunter/_Base/7d73237/4162dcb/76e14bd/Install/include/boost/exception/detail/error_info_impl.hpp:14,
                 from /root/.hunter/_Base/7d73237/4162dcb/76e14bd/Install/include/boost/exception/get_error_info.hpp:11,
                 from /root/.hunter/_Base/7d73237/4162dcb/76e14bd/Install/include/boost/exception/diagnostic_information.hpp:10,
                 from /root/.hunter/_Base/7d73237/4162dcb/76e14bd/Install/include/boost/exception/all.hpp:10,
                 from /root/.hunter/_Base/7d73237/4162dcb/76e14bd/Install/include/koinos/exception.hpp:4,
                 from /root/p/koinos/koinos-chain/libraries/vmmanager/include/koinos/vmmanager/exceptions.hpp:3,
                 from /root/p/koinos/koinos-chain/libraries/vmmanager/vm_backend.cpp:2:
/usr/include/c++/10/bits/basic_string.h:77:11: note: ‘std::__cxx11::basic_string<char>’ is not literal because:
   77 |     class basic_string
      |           ^~~~~~~~~~~~
/usr/include/c++/10/bits/basic_string.h:77:11: note:   ‘std::__cxx11::basic_string<char>’ has a non-trivial destructor
```